From 89a768926ceec7afb69c77cfbf7dad03184ae67b Mon Sep 17 00:00:00 2001
From: haiqiang yao <haiqiangx.yao@intel.com>
Date: Mon, 16 Nov 2015 09:59:00 +0800
Subject: [PATCH 18/28] atomisp.h: Add V4L2_CID_ATOMISP_SELECT_ISP_VERSION

the patch will add a new V4L2_CID_ATOMISP_SELECT_ISP_VERSION
in atomisp.h

Change-Id: I88cd6b8c2870ed8d0151eedaa018dbd9c204de42
Signed-off-by: haiqiang yao <haiqiangx.yao@intel.com>
Tracked-On: https://jira01.devtools.intel.com/browse/OAM-8630
Reviewed-on: https://android.intel.com:443/436572
---
 include/linux/atomisp.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/atomisp.h b/include/linux/atomisp.h
index 12640b9..3586546 100644
--- a/include/linux/atomisp.h
+++ b/include/linux/atomisp.h
@@ -1335,6 +1335,8 @@ struct atomisp_sensor_ae_bracketing_lut {
 #define V4L2_CID_TEST_PATTERN_COLOR_GB	(V4L2_CID_CAMERA_LASTP1 + 35)
 #define V4L2_CID_TEST_PATTERN_COLOR_B	(V4L2_CID_CAMERA_LASTP1 + 36)
 
+#define V4L2_CID_ATOMISP_SELECT_ISP_VERSION	(V4L2_CID_CAMERA_LASTP1 + 38)
+
 #define V4L2_BUF_FLAG_BUFFER_INVALID       0x0400
 #define V4L2_BUF_FLAG_BUFFER_VALID         0x0800
 
-- 
1.9.1

